test/box-luatest/gh_8051_set_box_cfg_thru_env_test.lua

```diff
@@ -1,47 +1,39 @@
+local fun = require('fun')
```

Comment:
No problem, will do at the next update.

Reply:
Updated the commit message and the pull request description with the following sentence.

> The test.interactive_tarantool helper is used for convenience instead of a popen wrapper that is written specifically for the test.